ajax/libs/primeng/package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "primeng",
+    "filename": "esm2015/primeng.js",
+    "description": "PrimeNG is a collection of rich UI components for Angular 2",
+    "version": "2.0.1",
```

Comment:
Please remove the version attr, the bot will populate this when it pulls in versioned files.

Reply:
```suggestion
```

That's ok with the new bot. I will move it once it lands